src/Components/Navbar/nav.jsx

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
 
+import { NavLink } from 'react-router-dom';
 import route from '../../route';
 
 import style from './nav.module.css';
 
+
 const NavBar = () => (
   <nav className={style.nav}>
     <div className={style.item}>
-      <a href={route.profile}>Profile</a>
+      <NavLink to={route.profile} activeClassName={style.active}>Profile</NavLink>
```

Comment:
Consider storing such strings as "Profile" in I18Y under Menu field - so that latter you could just go there and change the naming for the entire app.

Reply:
I will fix this problem next PR